### DISCUSSION TITLE
`FileNotFoundError: Source file does not exist .../site-packages/reactpy_router/static/bundle.jss`

### DISCUSSION THREAD

**Discussion by celsiusnarhwal** [reactive-python/reactpy-django#256] [Problem] [answered]
I'm not able to get started with ReactPy-Django because ReactPy-Router [appears to expect an internal file at `static/bundle.js`](https://github.com/reactive-python/reactpy-router/blob/7ffa76fa460a51ec24508d6dd068bd954b45d144/src/reactpy_router/components.py#L20) which [does not seem to exist](https://github.com/reactive-python/reactpy-router/tree/7ffa76fa460a51ec24508d6dd068bd954b45d144/src/reactpy_router/static). Manually creating an empty `bundle.js` file in the expected location makes the error go away, but I don't imagine that's what I'm supposed to do.

Given the absence of discussions and issues describing similar problems (I also checked the ReactPy and ReactPy-Router repositories and found nothing), I am inclined to believe there is something I am doing wrong, but cannot for the life of me figure out what it is.

## Steps to Reproduce

1. Create a new Django project.
2. Follow [ReactPy-Django's setup documentation](https://reactive-python.github.io/reactpy-django/latest/learn/add-reactpy-to-a-django-project/). The `FileNotFoundError` will occur when you try to execute `python manage.py migrate` as instructed in step 5. (Strictly speaking, it seems to occur when running `manage.py` *at all*, even without a command).

## Environment Details

**Python version:** 3.12.1
**Operating system:** macOS Sequoia 15.0.1
**ReactPy version:** 1.0.2
**ReactPy-Router version**: 1.0.1
**ReactPy-Django version**: 5.0.0

**Comment by Archmonger** [comment]
This might be the result of the reactpy-router `1.0.1` release from last night.

As a temporary fix, you can downgrade to reactpy-router `1.0.0`. I will work on resolving this issue today.

**Reply by Archmonger** [reply, marked as answer]
ReactPy Router version `1.0.2` has been released that includes the missing `bundle.js` file.
